### PR TITLE
JFWDisplay

### PR DESCRIPTION
**Pull request by SwareJonge**
code matches, data doesn't so that needs investigation

### CODE REVIEW

File: include/JSystem/JUT/JUTProcBar.h [hunk 1]
```diff
@@ -7,13 +7,61 @@
 class JUTProcBar {
 public:
 	struct CTime {
-		CTime();
+		u32 startTick;
```
Comment (intns) [inline include/JSystem/JUT/JUTProcBar.h:10]: Use proper naming schemes please. e.g. like 'm_example' or 'mExample'

**Review by intns — CHANGES_REQUESTED** [review]

File: include/JSystem/JUT/JUTProcBar.h (hunk 1 above)
Comment (EpochFlame) [reply include/JSystem/JUT/JUTProcBar.h]: going forward, we'll be using mExample consistently

File: include/JSystem/JFW/JFWDisplay.h [hunk 2]
```diff
@@ -51,26 +55,28 @@ struct JFWDisplay {
...
-	bool _24; // _24
-	u16 _26;  // _26
-	f32 _28;  // _28
-	int _2C;  // _2C
-	int _30;  // _30
-	int _34;  // _34
-	u16 _38;  // _38
-	u8 _3A;   // _3A
+	u32 m_tickRate;         // _20
+	bool m_enableAlpha;     // _24
```
Comment (intns) [inline include/JSystem/JFW/JFWDisplay.h:72]: When naming booleans, please make sure we're asking the question, in this example it'll be something like `mIsAlphaEnabled`

File: include/JSystem/JUT/JUTProcBar.h [hunk 3]
```diff
@@ -7,13 +7,77 @@
...
-		u32 _0C;   // _0C
-		u8 _10[4]; // _10 - unknown
+		void clear()
+		{
+			mCost = 0;
+			_08   = 0;
+			_0C   = 0;
+		}
+
+		void start(u8 p1, u8 p2, u8 p3)
```
Comment (intns) [inline include/JSystem/JUT/JUTProcBar.h:27]: Please name the arguments appropriately

File: include/JSystem/JUT/JUTProcBar.h [hunk 4]
```diff
@@ -7,13 +7,77 @@
...
+			mR         = p1;
+			mG         = p2;
+			mB         = p3;
+			mStartTick = OSGetTick();
+		}
+
+		void end()
+		{
+			mCost = OSTicksToMicroseconds(OSDiffTick(OSGetTick(), mStartTick));
+			if (mCost == 0)
```
Comment (intns) [inline include/JSystem/JUT/JUTProcBar.h:38]: Close the brackets in small if statements, regardless

File: include/JSystem/JUT/JUTProcBar.h [hunk 5]
```diff
@@ -7,13 +7,77 @@
...
+
+		void accumePeek()
+		{
+			if (++_0C >= 0x10 || mCost >= _08) {
+				_08 = mCost;
+				_0C = 0;
+			}
+		}
+
+		int calcBarSize(int p1, int p2) // fabricated
```
Comment (intns) [inline include/JSystem/JUT/JUTProcBar.h:50]: please name the arguments, if you know them

File: include/JSystem/JUT/JUTProcBar.h [hunk 6]
```diff
@@ -7,13 +7,77 @@
...
+		void setBarWidth(int w) { mBarWidth = w; };
+		void setWidth(int w) { mWidth = w; }
+		void setUserPosition(int pos) { mUserPosition = pos; }
+		void setPosition(int x, int y)
+		{
+			mPosX = x;
+			mPosY = y;
+		}
+
+		/* 0x00 */ int mBarWidth;
```
Comment (intns) [inline include/JSystem/JUT/JUTProcBar.h:76]: Please use the standards we've set regarding offset comments

File: include/JSystem/JUT/JUTProcBar.h [hunk 7]
```diff
@@ -31,42 +95,49 @@ class JUTProcBar {
 	void adjustMeterLength(u32, f32*, f32, f32, int*);
 	void getUnuseUserBar();
 
+	inline u32 calcGPUTime()
+	{ // fabricated
```
Comment (intns) [inline include/JSystem/JUT/JUTProcBar.h:99]: no need to add fabricated inline comments

File: include/JSystem/JUT/JUTXfb.h [hunk 8]
```diff
@@ -45,14 +45,38 @@ struct JUTXfb {
...
+
+	void *getDrawingXfb() const {
+	    return (mDrawingXfbIndex >= 0) ? mBuffer[mDrawingXfbIndex] : nullptr;
+	}
+
+	void *getDisplayingXfb() const {
+	    return (mDisplayingXfbIndex >= 0) ? mBuffer[mDisplayingXfbIndex] : nullptr;
+	}*/
+
+	EXfbNumber getBufferNum() const { return _10; }
```
Comment (intns) [inline include/JSystem/JUT/JUTXfb.h:60]: Please rename the variables appropriately

**Review by intns — CHANGES_REQUESTED** [review]

**Pull request merged**